k8/deploy/postgres-db-deploy.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
```

Comment:
note that in a real production system, you would want to use something like
https://github.com/zalando/postgres-operator

For us, below is fine.